mpact/models.py

```diff
@@ -30,21 +30,33 @@ def __str__(self):
 
 class Bot(models.Model):
     chat = models.OneToOneField(Chat, on_delete=models.CASCADE)
-    bot_id = models.IntegerField(unique=True)
+    bot_id = models.IntegerField()
     username = models.CharField(max_length=350)
     first_name = models.CharField(max_length=350)
     last_name = models.CharField(max_length=350, null=True)
 
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["chat", "bot_id"], name="unique_chat_bot")
+        ]
+
     def __str__(self):
         return f"{self.bot_id} - {self.username}"
 
 
 class Individual(models.Model):
     bot = models.ForeignKey(Bot, related_name="individuals", on_delete=models.CASCADE)
-    individual_id = models.IntegerField(unique=True)
+    individual_id = models.IntegerField()
     username = models.CharField(max_length=350, null=True)
     first_name = models.CharField(max_length=350)
     last_name = models.CharField(max_length=350)
 
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(
+                fields=["bot", "individual_id"], name="unique_bot_individuals"
```

Comment:
Same question as for `Bot` model. Keep `individual_id` as unique, so you are not storing duplicate records for the same person, and change ` bot` to a `MantToMany`?

Reply:
As we discussed yesterday if there is the same bot in two groups and user direct messages the bot. Then we get two records of the bot with the G1 and G2 groups. Now we need to map user details with both the records and show in both the accordion. That is the reason I was storing multiple records.